db/image_job.sql

```diff
@@ -0,0 +1,54 @@
+CREATE TABLE IF NOT EXISTS IMAGE_JOB(
+    job_id  BIGINT NOT NULL REFERENCES JOB(job_id),
+    /*job_status INTEGER NOT NULL DEFAULT 0,*/
+    CONSTRAINT pk_image_job PRIMARY KEY(job_id)
+);
+
+CREATE TABLE IF NOT EXISTS IMAGE_STATUS(
+    imstat_id    INTEGER CONSTRAINT pk_imstat PRIMARY KEY,
+    imstat_name  TEXT NOT NULL
+);
+
+INSERT INTO IMAGE_STATUS(imstat_id, imstat_name) VALUES
+    (0, 'WAITING'),
+    (1, 'PREPROCESSING'),
+    (2, 'PREPROCESSED'),
+    (3, 'PROCESSING'),
+    (4, 'PROCESSED');
+
+/*IMAGE may be a reserved name*/
+CREATE TABLE IF NOT EXISTS IMAGE(
+    image_id        BIGINT GENERATED ALWAYS AS IDENTITY,
```

Comment:
Is this necessary if we have the path? No two images should have the same path

Reply:
It's bad practice to have variable length strings as primary key.